vite.config.js

```diff
@@ -29,6 +29,9 @@ export default defineConfig(({ mode }) => {
           'resources/css/app.css',
           'resources/js/app.js',
         ],
+        refresh: [
+          'resources/views/**'
+        ]
```

Comment:
Vite was not reloading on edits to blades, making working with Tailwind kind of frustrating. This fixed the issue.

Reply:
This used to be in there but was removed to some performance problems with chokidar which tested too many files. Has to be tested within a Linux VM again to make sure the problem didn't come back

Reply:
I can look into setting this to be enabled if an env var is truthy, and set that new var to be falsy by default. That way there's an easy way to toggle it, and it's documented why it's disabled by default.

Reply:
I don't think an additional env var is necessary - given that it just works in a VM. @Jamiras if you find time, could you please check if the change in the vite config causes any of the previous problems in your VM?

If it doesn't then it can stay imo. Unfortunately I haven't found time to set up a VM on my windows machine yet.

Reply:
Seems to be okay. There's only 124 files in `resources/views`. My problem was files in storage, where I have over 60000.

Reply:
Perfect, thanks for checking!